patterns/contact-info-locations.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Title: Contact: Info and locations
+ * Slug: twentytwentyfive/contact-info-locations
+ * Keywords: contact, location
+ * Categories: contact
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"var:preset|spacing|50","left":"var:preset|spacing|50","top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"}}},"layout":{"type":"default"}} -->
```

Comment:
I believe the pattern should take a max-width of site wide, and it's taking full width.